src/actions/guides/http_and_routing/routing_and_params.cr

```diff
@@ -179,7 +179,7 @@ class Guides::HttpAndRouting::RoutingAndParams < GuideAction
 
     ### Namespaces are handled automatically
 
-    You can namespace your actions by creating subfolders like `src/actions/admin/projects/index.cr`.
+    You can namespace your routes by prefixing the class name, e.g. with `Admin::`.
```

Comment:
Would be good to put the filename on line 185 as a comment I think: `# in src/actions/admin/projects/index.cr`